modules/server/main.tf

```diff
@@ -72,243 +80,244 @@ locals {
-    KE_VERSION_VIOLATIONS_SHOULD_NOTIFY                 = "${var.ke_version_violations_should_notify ? "true" : "false"}"
-    KMS_VIOLATIONS_SHOULD_NOTIFY                        = "${var.kms_violations_should_notify ? "true" : "false"}"
-    KMS_VIOLATIONS_SLACK_WEBHOOK                        = "${var.kms_violations_slack_webhook}"
-    INVENTORY_GCS_SUMMARY_ENABLED                       = "${var.inventory_gcs_summary_enabled ? "true" : "false"}"
-    INVENTORY_EMAIL_SUMMARY_ENABLED                     = "${var.inventory_email_summary_enabled ? "true" : "false"}"
-    INSTANCE_NETWORK_INTERFACE_VIOLATIONS_SHOULD_NOTIFY = "${var.instance_network_interface_violations_should_notify ? "true" : "false"}"
-    IAP_VIOLATIONS_SHOULD_NOTIFY                        = "${var.iap_violations_should_notify ? "true" : "false"}"
-    IAM_POLICY_VIOLATIONS_SHOULD_NOTIFY                 = "${var.iam_policy_violations_should_notify ? "true" : "false"}"
-    IAM_POLICY_VIOLATIONS_SLACK_WEBHOOK                 = "${var.iam_policy_violations_slack_webhook}"
-    GROUPS_VIOLATIONS_SHOULD_NOTIFY                     = "${var.groups_violations_should_notify ? "true" : "false"}"
-    FORWARDING_RULE_VIOLATIONS_SHOULD_NOTIFY            = "${var.forwarding_rule_violations_should_notify ? "true" : "false"}"
-    FIREWALL_RULE_VIOLATIONS_SHOULD_NOTIFY              = "${var.firewall_rule_violations_should_notify ? "true" : "false"}"
-    EXTERNAL_PROJECT_ACCESS_VIOLATIONS_SHOULD_NOTIFY    = "${var.external_project_access_violations_should_notify ? "true" : "false"}"
-    ENABLED_APIS_VIOLATIONS_SHOULD_NOTIFY               = "${var.enabled_apis_violations_should_notify ? "true" : "false"}"
-    CLOUDSQL_ACL_VIOLATIONS_SHOULD_NOTIFY               = "${var.cloudsql_acl_violations_should_notify ? "true" : "false"}"
-    CONFIG_VALIDATOR_VIOLATIONS_SHOULD_NOTIFY           = "${var.config_validator_violations_should_notify ? "true": "false"}"
-    BUCKETS_ACL_VIOLATIONS_SHOULD_NOTIFY                = "${var.buckets_acl_violations_should_notify ? "true" : "false"}"
-    BLACKLIST_VIOLATIONS_SHOULD_NOTIFY                  = "${var.blacklist_violations_should_notify ? "true" : "false"}"
-    BIGQUERY_ACL_VIOLATIONS_SHOULD_NOTIFY               = "${var.bigquery_acl_violations_should_notify ? "true" : "false"}"
-    AUDIT_LOGGING_VIOLATIONS_SHOULD_NOTIFY              = "${var.audit_logging_violations_should_notify ? "true" : "false"}"
-
-    VIOLATIONS_SLACK_WEBHOOK                            = "${var.violations_slack_webhook}"
-
+  vars = {
+    ROOT_RESOURCE_ID                                    = local.root_resource_id
+    COMPOSITE_ROOT_RESOURCES                            = local.composite_root_resources
+    DOMAIN_SUPER_ADMIN_EMAIL                            = var.gsuite_admin_email
+    CAI_ENABLED                                         = var.enable_cai_bucket ? "true" : "false"
```

Comment:
The boolean assignments can all be simplified like the following since booleans will be interpolated as strings:

```
CAI_ENABLED = var.enable_cai_bucket
```

Reply:
Thank you, I changed construction to more simple